frontend/src/routes/Infrastructure/InfraEnvironments/utils.ts

```diff
@@ -1,5 +1,12 @@
 /* Copyright Contributors to the Open Cluster Management project */
+import { isMatch } from 'lodash'
 import { CIM } from 'openshift-assisted-ui-lib'
 
 export const isBMPlatform = (infrastructure?: CIM.InfrastructureK8sResource) =>
-    ['BareMetal', 'None', 'OpenStack', 'VSphere'].includes(infrastructure?.status?.platform)
+    ['BareMetal', 'None', 'OpenStack', 'VSphere', 'AWS'].includes(infrastructure?.status?.platform)
```

Comment:
AWS sneaking in again!

```suggestion
    ['BareMetal', 'None', 'OpenStack', 'VSphere'].includes(infrastructure?.status?.platform)
```

Reply:
Done